docs/docs/tutorial.md

```diff
@@ -374,7 +374,7 @@ var CommentBox = React.createClass({
 `getInitialState()` executes exactly once during the lifecycle of the component and sets up the initial state of the component.
 
 #### Updating state
-When the component is first created, we want to GET some JSON from the server and update the state to reflect the latest data. In a real application this would be a dynamic endpoint, but for this example, we will use a static JSON file to keep things simple:
+When the component is first created, we want to GET some JSON from the server and update the state to reflect the latest data. In a real application this would be a dynamic endpoint, but for this example we will keep things simple by creating a static JSON file public/comments.json containing the array of comments:
 
 ```javascript
 // tutorial13.json
```

Comment:
The tutorial is broken into steps with file names to match (eg `tutorial13.js` would not actually be named that on disk, just used to indicate the step of the tutorial). Honestly we could probably get rid of those comments entirely.